google/resource_spanner_database.go

```diff
@@ -485,6 +500,13 @@ func flattenSpannerDatabaseInstance(v interface{}, d *schema.ResourceData, confi
 	return ConvertSelfLinkToV1(v.(string))
 }
 
+func flattenSpannerDatabaseDialect(v interface{}, d *schema.ResourceData, config *Config) interface{} {
+	if v == nil {
+		return v
+	}
+	return ConvertSelfLinkToV1(v.(string))
```

Comment:
`ConvertSelfLinkToV1()` cleans URLs.  But database dialects are enums, I didn't think they can contain URLs?  So that function should be a no-op for any valid dialect.  Should this be a no-op like (for example) `flattenSpannerDatabaseState()` above?